src/ui/src/utils/string-search.ts

```diff
@@ -183,18 +183,23 @@ export function highlightScoredMatch(search: string, source: string): ScoredStri
     // Exact prefix is even better than exact substring, but not as good as a perfect match
     return {
       isMatch: true,
-      distance: Math.abs(searchNorm.length - sourceNorm.length) / 4,
+      distance: Math.abs(searchNorm.length - sourceNorm.length) / 5,
```

Comment:
Is the 5 a magic number? How did you decide on it?

Reply:
Extracted the magic numbers.